source/PlayerInfo.cpp

```diff
@@ -2340,6 +2340,15 @@ void PlayerInfo::StepMissions(UI *ui)
 				if(ship->IsDestroyed())
 					mission.Do(ShipEvent(nullptr, ship, ShipEvent::DESTROY), *this, ui);
 	
+	string visitText;
+	int extraVisitDialogs = 0;
+	int deadlineMissions = 0;
+	map<string, string> subs;
```

Comment:
Initialize this map with the name substitutions

Reply:
sorry, i meant initialize it with the `<first>` and `<last>` substitutions, not to rename it 😅 
e.g.
```c++
auto subs = map<string, string>{
  {"<first>", firstName},
  {"<last>", lastName}
};
```